index.html

```diff
@@ -304,68 +419,349 @@
+                    <use href="#flag-icons-us"></use>
+                  </svg>
+                  <p class="rev__reviews-user-country">USA</p>
+                </div>
+              </div>
+            </div>
+            <p class="rev-reviews">
+              I've been in this hostel for a month and here is the best hostel
+              of all <br>
+              Excellent management of excellent and ethical staff and
+              bedrooms Bathrooms are always clean and excellent <br>
+              Most
+              importantly, those who work here are kind and good-natured.
+            </p>
+          </div>
+          <div class="col-lg-4 col-md-6 col-sm-6 rev__row-elem">
+            <div class="rev__image">
+              <img
+                src="./images/reviews-image/hotel-reviews.png"
+                alt="hotel"
+                class="rev__image"
+              />
+              <div class="rev__price">From 100 USD</div>
+            </div>
+            <div class="rev__info">
+              <div class="rev__info-left">
+                <p class="rev__info-title">Rokoko Hotel</p>
+                <p class="rev__info-place">Ourika, Marocco</p>
+              </div>
+              <div class="rev__info-right">
+                <p class="rev__info-grade">9,1</p>
+                <p class="rev__info-count">815 reviews</p>
+              </div>
+            </div>
+            <div class="rev__account">
+              <div class="rev__icon">
+                <svg class="rev__icon-account">
+                  <use href="#rev-icon-account" />
+                </svg>
+              </div>
+              <div class="rev__reviews-user">
+                <p class="rev__reviews-user-name">Anika Messer</p>
+                <div class="rev__reviews-user-country">
+                  <svg class="rev__country-icon">
+                    <use href="#flag-icons-de"></use>
+                  </svg>
+                  <p class="rev__reviews-user-country">Germany</p>
+                </div>
+              </div>
+            </div>
+            <p class="rev-reviews">
+              Efficient, friendly, professional, and appropriately attentive.
+              Great hotel in a great location. Easy access to all the
+              surrounding historical sites. Staff is fantastic. Will definitely
+              stay at this location during my next stay in Marocco.
+            </p>
+          </div>
+          <svg class="rev_row-arrow">
+            <use href="#arrow" />
+          </svg>
         </div>
-        <div class="col-lg-3 col-md-4 col-sm-3 homes__row-elem">
-          <img class="homes__image" src="images/homes-images/villa.png" alt="villa">
-          <a class="homes__types-title" href="#">Villa Kunerad</a>
-          <p class="homes__row-text">Vysokie Tatry, Slowakia</p>
+        <div action="#" class="rev__switch-mob--hidden">
+          <button type="button" class="rev__switch-mob" autofocus>
```

Comment:
A document must not include more than one 'autofocus' attribute (see the row 509).